README.md

```diff
@@ -391,6 +391,8 @@ Render all underlying links as follows:
 - Internal links use client-side navigation with smooth scrolling via [react-router-hash-link],
   and pass through the `v` URL parameter for UI version switching
 - External links open in a new tab
+- Attachment links open attachments in a new tab.
+  Link components must set `datatype` attribute to `attachments`
```

Comment:
1. What's `datatype`? A cursory search seems to tell me that this is a non-standard attribute that has no usage internally at SEEK.
2. Would this be better as a singular `attachment` or maybe even `download`?

Reply:
1. Couldn't find nice attributes to use that are typed in `makeLinkComponent` 
2. Yep download sounds good. But just noting that for pdf case, the pdf opens in browser so doesn't feel download-y

Reply:
How about this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download